library/src/main/java/org/jellyfin/apiclient/IDevice.kt

```diff
@@ -1,4 +1,4 @@
-package org.jellyfin.apiclient.interaction.device
+package org.jellyfin.apiclient
 
 interface IDevice {
```

Comment:
Can we ditch the “I” prefix since we’re changing things?

Reply:
I'm going to do that when implementing the new API. This interface might be removed entirely.